pyroengine/engine.py

```diff
@@ -375,7 +354,7 @@ def _process_alerts(self) -> None:
             logging.info(f"Camera '{cam_id}' - Sending alert from {frame_info['ts']}...")
 
             # Save alert on device
-            self._local_backup(frame_info["frame"], cam_id, is_alert=True)
+            self._local_backup(frame_info["frame"], cam_id)
```

Comment:
maybe we should remove the backup file once the media have been saved ?

Reply:
yes ! I will make an update soon, we same all alerts localy and once it's uploaded to the api we remove it